nzediegwu1/blog-connect-web
# #162181998 Setup create-react-app

#### What does this PR do?
Integrate and setup create-react-app for UI development

#### Description of Task to be completed?
- Install Create-react-app
- Re-arranged folder structure in the react app generated


#### How should this be manually tested?
- Running `yarn start` command on terminal should start the app on browser
- Running `yarn test` on terminal should run tests, which should pass

#### Any background context you want to provide?
Create-react-app eliminates the stress of:
- Configuring webpack for bundling css, javascript and images, minification and production setups
- Installing and setting-up babel and dependencies for ES6+
- Setting up HMR for react development
- Setting up testing framework, among others

#### What are the relevant pivotal tracker stories?
[#162181998](https://www.pivotaltracker.com/story/show/162181998)

#### Screenshots (If applicable)
N/A
